docs/myriad-configuration.md

```diff
@@ -63,10 +69,14 @@ executor:
   jvmMaxMemoryMB: 256   # Xmx for myriad's executor that launches Node Manager.
   path: file://localhost/usr/local/libexec/mesos/myriad-executor-0.0.1.jar  # Path for the myriad's executor binary.
                                                                             # Also supports, hdfs:// notation.
+  #These are for remote distribution
+  #path: hadoop-2.5.0/share/hadoop/yarn/lib/myriad-executor-0.0.1.jar #this should be relative if nodeManagerUri is set
+  #nodeManagerUri: hdfs://namenode:port/dist/hadoop-2.5.0.tar.gz # the uri to d/l hadoop from
```

Comment:
Yes I assume the executor is bundled in the tarball.  This seems to be the approach of hadoop and storm.  I'm open for making it a separate uri.  I'm actually grabbing the hadoop configs from the resource manager using a separate uri.